performance-tests/src/test/java/org/eclipse/collections/impl/parallel/SerialParallelPerformanceTest.java

```diff
@@ -224,7 +224,7 @@ private void measureAlgorithmForRandomStringIterable(
             Procedure<Function0<Iterable<String>>> algorithm)
     {
         this.printMachineAndTestConfiguration(algorithmName);
-        for (int i = 0; i < 4; i++)
+        for (int i = 0; i < 1; i++)
```

Comment:
Comments here also.

Reply:
Just added some comments -- can you take a look? Thanks!